sdk/resourcemanager/Azure.ResourceManager.Core/src/Generated/Tags.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.ResourceManager.Core
+{
+    /// <summary>
+    /// A class representing a Tags along with the instance operations that can be performed on it.
+    /// </summary>
+    public class Tags : TagsOperations
```

Comment:
use singular name for resource Tag and all related files

Reply:
Done